demo/genome_config.yaml

```diff
@@ -2,22 +2,56 @@ lcr-modules:
 
     _shared:
         lcr-modules: "../"
-        lcr-scripts: "../../lcr-scripts/"
+        lcr-scripts: "../../lcr-scripts"
```

Comment:
Best to leave the trailing slash at the end of the directory name, unless there is a specific reason not to. If so please document that reason.

Reply:
done